### PR TITLE
COR-1489-remove-toggle-view-on-patients-page

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
## Summary

* 
* removal of the safety region choropleth on the national 'View on patients'-page
*
*

#### Before
<details>
<summary>Toggle before screenshots</summary>

Screenshot 2023-03-28 at 17 27 18

</details>

#### After
<details>
<summary>Toggle after screenshots</summary>

Screenshot 2023-03-28 at 17 27 10

</details>

### CODE REVIEW

**Review by VWSCoronaDashboard27 — DISMISSED** [review]

**Comment by VWSCoronaDashboard27** [comment]
Looks good for me!

File: packages/app/src/pages/landelijk/patienten-in-beeld.tsx [hunk 1]
```diff
@@ -145,56 +140,32 @@ const PatientsPage = (props: StaticProps<typeof getStaticProps>) => {
...
-            onChartRegionChange={setSelectedMap}
-            chartRegion={selectedMap}
             legend={{
-              thresholds: selectedMap === 'gm' ? thresholds.gm.admissions_on_date_of_admission_per_100000 : thresholds.gm.admissions_on_date_of_admission_per_100000,
+              thresholds: thresholds.gm.admissions_on_date_of_admission_per_100000,
               title: textNl.choropleth.legend_title,
             }}
             metadata={{
-              date: lastValueNiceChoropleth.date_unix,
+              date: lastValueNice.date_unix,
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/pages/landelijk/patienten-in-beeld.tsx:148]: Is this change intended?

I think we should alter and use `lastValueNiceChoropleth` that was removed from line 107 as there could be discrepancies between `lastValueNice` and `lastValueNiceChoropleth`. This can also already be noticed on the [production environment](https://coronadashboard.rijksoverheid.nl/landelijk/patienten-in-beeld).

Comment (VWSCoronaDashboard18) [reply packages/app/src/pages/landelijk/patienten-in-beeld.tsx]: It's been adjusted to the correct usage. Please check and confirm. If you agree. Please resolve.

**Review by VWSCoronaDashboard21 — APPROVED** [review]
Looks great!

**Review by VWSCoronaDashboard26 — APPROVED** [review]
